ubg/package.json

```diff
@@ -15,6 +15,7 @@
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-elastic-carousel": "^0.6.7",
+    "react-hook-form": "^6.3.0",
```

Comment:
Where are you using this?

Reply:
I was using it for validation and switched techniques. Will uninstall.